app/src/components/pages/collectionsPage/collectionsPage.tsx

```diff
@@ -37,20 +146,111 @@ export const CollectionsPage = ({ data }) => {
           subtitle="Explore the New York Public Library's diverse collections, including digitized photographs, manuscripts, maps, and more. Start exploring by using the search bar below or browse through the collections."
         />
         <SearchBar
-          sx={{ maxWidth: "462px", marginTop: "l" }}
+          sx={{ maxWidth: "462px" }}
           id={"search-collections"}
           textInputProps={{
             isClearable: true,
+            isClearableCallback: () => {
+              setCurrentCollectionKeyword(DEFAULT_SEARCH_TERM);
+            },
             labelText: "Search by collection title",
-            name: "searchinput",
+            name: "collection_keyword",
             placeholder: "Search by collection title",
+            defaultValue: currentCollectionKeyword,
+            onChange: (e) => handleSearchChange(e),
           }}
-          onSubmit={function (event: React.FormEvent): void {}}
+          onSubmit={handleSearchSubmit}
           labelText={""}
         />
       </Box>
       <HorizontalRule sx={{ marginTop: "xxl", marginBottom: "xxl" }} />
-      <SearchResults showFilter isSearchPage={false} data={data} />
+      <Box
+        sx={
+          collections?.length > 0
+            ? { display: "flex", gap: "xs", marginBottom: "l" }
+            : { display: "none", gap: "xs", marginBottom: "l" }
+        }
+      >
+        <Text sx={{ fontWeight: "500", marginBottom: 0, marginTop: "xs" }}>
+          {" "}
+          Sort by{" "}
+        </Text>{" "}
+        <Menu
+          showSelectionAsLabel
+          showLabel
+          selectedItem={currentSort}
+          labelText={"Sort By"}
+          listItemsData={[
+            {
+              id: "date-desc",
+              label: "Newest to oldest",
+              onClick: onMenuClick,
+              type: "action",
+            },
+            {
+              id: "date-asc",
+              label: "Oldest to newest",
+              onClick: onMenuClick,
+              type: "action",
+            },
+            {
+              id: "title-asc",
+              label: "Title A to Z",
+              onClick: onMenuClick,
+              type: "action",
+            },
+            {
+              id: "title-desc",
+              label: "Title Z to A",
+              onClick: onMenuClick,
+              type: "action",
+            },
+          ]}
+        />
+      </Box>
+      <Heading
+        size="heading5"
+        sx={
+          collections?.length > 0
+            ? { marginBottom: "l" }
+            : { display: "none", marginBottom: "l" }
+        }
+        ref={headingRef}
+        tabIndex={-1}
+        id={"all-collections-page"}
```

Comment:
nit: you dont need braces around these types of strings

Reply:
not a nit, this is evidence of bad practice